boards/esp32-c3-devkitM-1/src/main.rs

```diff
@@ -169,10 +182,42 @@ unsafe fn setup() -> (
     // Configure kernel debug gpios as early as possible
     kernel::debug::assign_gpios(None, None, None);
 
+    //
+    // UART
+    //
+
     // Create a shared UART channel for the console and for kernel debug.
     let uart_mux = components::console::UartMuxComponent::new(&peripherals.uart0, 115200)
         .finalize(components::uart_mux_component_static!());
 
+    // Setup the console.
+    let console = components::console::ConsoleComponent::new(
+        board_kernel,
+        capsules_core::console::DRIVER_NUM,
+        uart_mux,
+    )
```

Comment:
Is this PR blocked on some other PR first? I'm not sure why the console changes are part of this.

Reply:
Doesn't seem to include any console-related commits, so this seems like a mistake on the surface.

Reply:
The main.rs was in a confusing order with no sections. I feel that part of adding more syscall drivers is reorganizing main.rs so logical inits are grouped and labeled.